lib/OpenQA/Utils.pm

```diff
@@ -417,6 +419,12 @@ sub href_to_bugref {
     return $text;
 }
 
+sub url_to_href {
+    my ($text) = @_;
+    $text =~ s(($RE{URI}))(<a href="$1">$1</a>)gx;
+    return $text;
+}
```

Comment:
Seems to be two different things. Here I want to replace *any* URL with corresponding `a` tag. In `Comments.pm` only URLs which aren't already in HTML tags or Markdown are replaced with Markdown syntax.